package.json

```diff
@@ -17,7 +17,7 @@
     "prepare": "husky install",
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "cypress:headed": "cypress run --headed"
+    "update-types": "dotenv -e .env -- bash -c 'npx supabase gen types --lang=typescript --project-id $PROJECT_ID --schema public > supabase/types.ts'"
```

Comment:
Thank you for this. But please make it cross platform (this would not work outside of bash environments). Please refer to `cross-env` or equivalent.